pkg/release-controller/release_info.go

```diff
@@ -53,8 +50,8 @@ func NewCachingReleaseInfo(info ReleaseInfo, size int64, architecture string) Re
 				var bugDetailsArr []BugDetails
 				bugDetailsArr, err = info.Bugs(parts[1], parts[2])
 				if err == nil {
-					bugDetailsByte, err := json.Marshal(bugDetailsArr)
-					if err != nil {
+					bugDetailsByte, parseErr := json.Marshal(bugDetailsArr)
+					if parseErr != nil {
```

Comment:
So, if there is an error parsing, and you're not setting `err` anymore, but still checking for `err` down below (Line 79), what's getting cached?

Reply:
Instead of using `parseErr`, I changed it to:
```golang
var bugDetailsByte []byte
bugDetailsByte, err = json.Marshal(bugDetailsArr)
```

That should fix the `err` shadowing.